CONTRIBUTING.md

```diff
@@ -52,6 +52,11 @@ cargo test
 
 This command will be triggered to each PR as a requirement for merging it.
 
+#### Faster build
+
+You can set the `LINDERA_CACHE` environment variable to speed up your successive builds by up to 2 minutes.
```

Comment:
Can you add more info about what you must put in this variable, that the folder shouldn't necessarily exist, and that a default good value is `$HOME/.cache/lindera`?